test/cluster/test_tablet_snapshot.py

```
@@ -0,0 +1,219 @@
+    """
+    snapshot_script = TOP_SRC_DIR / "scripts" / "tablets" / "snapshot.py"
+    result = subprocess.run(
+        [sys.executable, str(snapshot_script), "--manual", "--cluster", host, "--output-dir", output_dir],
+        cwd=workdir,
+        check=True,
+        capture_output=True,
+        text=True,
+    )
+    for line in result.stdout.strip().splitlines():
+        if line.startswith("cqlsh "):
+            line = line.replace(
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)